modules/understanding-process-id-limits.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-configuring-pid-limits.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="understanding-process-id-limits_{context}"]
+= Understanding process ID limits
+
+A process identifier (PID) is a unique identifier assigned by the Linux kernel to each process or thread currently running on a system. The number of processes that can run simultaneously on a system is limited to 4,194,304 by the Linux kernel, and may additionally be affected by limited access to other system resources such as memory, CPU, and disk space.
+
+In {product-title}, there are two supported limits for PID usage to consider when you schedule work on your cluster:
+
+Maximum number of PIDs per pod (default: 4,096 in {product-title} 4.11 and higher)::
+This is controlled by the `podPidsLimit` parameter set on the node.
+
+Maximum number of PIDs per node (default depends on link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html-single/nodes/index#nodes-nodes-resources-configuring[node resources])::
+In OpenShift, this is controlled by the link:https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#system-reserved[`--system-reserved`] parameter, which reserves PIDs on each node based on the total resources of the node.
```

Comment:
@bburt-rh, Is the formatting okay? I think the writer wants to highlight these points using a description list.

Reply:
@shipsing @bhardesty - The two items here should be formatted as a bulleted list.

Reply:
```suggestion
Maximum number of PIDs per pod (default: 4,096 in {product-title} 4.11 and higher)::
This is controlled by the `podPidsLimit` parameter set on the node.

* Maximum number of PIDs per node. The default value depends on link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html-single/nodes/index#nodes-nodes-resources-configuring[node resources]).
In {product-title}, this value is controlled by the link:https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#system-reserved[`--system-reserved`] parameter, which reserves PIDs on each node based on the total resources of the node.
```
- Format as a bullet point.
- Replace "OpenShift" with the correct attribute.
- Rewrite to avoid ambiguous pronoun reference.
- Also, as a point for future reference: Generally in the non-ROSA OCP docs, we don't cross-reference other OCP docs by using direct links the published HTML; we should always use xrefs to adoc files, though these xrefs can't be used in module files. However, for the ROSA docs, there's an exception because we can't link between different distros, such as from ROSA to OpenShift Container Platform. Thus, these can be external links and not cross-references. These external links use `link` instead of `xref` and need to be in distro-specific files or conditionalized to apply to the relevant distros.

Reply:
FYI, re: the link to "node resources" - I changed the module file name to start with the `sd-` prefix. The reason for the link is because it points to a topic that is not included in the ROSA docs. Because we don't have a good way to handle these scenarios, we typically just link to the OCP docs on the customer portal. I renamed the file to indicate that this is a Managed OpenShift (i.e. "service delivery") module, and is not intended for OCP.